php/libraries/FeedbackMRI.class.inc

```diff
@@ -202,7 +201,7 @@ class FeedbackMRI
         $DB =& Database::singleton();
 
         // build the query
-        $query = "SELECT PredefinedCommentID, Comment, FieldName
+        $query = "SELECT PredefinedCommentID, Comment
```

Comment:
FieldName column does not exist in the table, this was throwing an exception when select was executed